src/improvements/justfile

```diff
@@ -125,7 +125,7 @@ monorepo-integration-test COMMAND="":
Pause_Test\|L1ERC721Bridge_Pause_TestFail\|L1ERC721Bridge_Test\|L1StandardBridge_BridgeETHTo_Test\|L1StandardBridge_BridgeETH_Test\|L1StandardBridge_DepositERC20To_Test\|L1StandardBridge_DepositERC20_Test\|L1StandardBridge_DepositERC20_TestFail\|L1StandardBridge_DepositETHTo_Test\|L1StandardBridge_DepositETH_Test\|L1StandardBridge_DepositETH_TestFail\|L1StandardBridge_FinalizeBridgeETH_Test\|L1StandardBridge_FinalizeBridgeETH_TestFail\|L1StandardBridge_FinalizeERC20Withdrawal_Test\|L1StandardBridge_FinalizeERC20Withdrawal_TestFail\|L1StandardBridge_FinalizeETHWithdrawal_Test\|L1StandardBridge_Getter_Test\|L1StandardBridge_Initialize_Test\|L1StandardBridge_Pause_Test\|L1StandardBridge_Pause_TestFail\|L1StandardBridge_Receive_Test\|OptimismPortal2_FinalizeWithdrawal_Test\|OptimismPortal2_LiquidityMigration_Test\|OptimismPortal2_ResourceFuzz_Test\|OptimismPortal2_Test\|OptimismPortal2_Upgradeable_Test\|OptimismPortal2_upgrade_Test\|ProtocolVersions_Initialize_Test\|ProtocolVersions_Setters_Test\|ProtocolVersions_Setters_TestFail\|StandardValidatorV180_Test\|SuperchainConfig_Init_Test\|SuperchainConfig_Pause_Test\|SuperchainConfig_Pause_TestFail\|SuperchainConfig_Unpause_Test\|SuperchainConfig_Unpause_TestFail\|SystemConfig_Init_ResourceConfig\|SystemConfig_Initialize_Test\|SystemConfig_Initialize_TestFail\|SystemConfig_Setters_Test\|SystemConfig_Setters_TestFail\|SystemConfig_upgrade_Test\|MIPS_Test\|MIPS2_Test\|MIPS64Memory_Test\|AnchorStateRegistry_Anchors_Test\|AnchorStateRegistry_GetAnchorRoot_Test\|AnchorStateRegistry_Initialize_Test\|AnchorStateRegistry_Initialize_TestFail\|AnchorStateRegistry_IsGameBlacklisted_Test\|AnchorStateRegistry_IsGameClaimValid_Test\|AnchorStateRegistry_IsGameFinalized_Test\|AnchorStateRegistry_IsGameProper_Test\|AnchorStateRegistry_IsGameRegistered_Test\|AnchorStateRegistry_IsGameResolved_Test\|AnchorStateRegistry_IsGameRespected_Test\|AnchorStateRegistry_IsGameRetired_Test\|AnchorStateRegistry_Paused_Test\|AnchorStateRegistry_SetAnchorState_Test\|AnchorStateRegistry_SetAnchorState_TestFail\|AnchorStateRegistry_Version_Test\|AnchorStateRegistry_blacklistDisputeGame_Test\|AnchorStateRegistry_blacklistDisputeGame_TestFail\|AnchorStateRegistry_setRespectedGameType_Test\|AnchorStateRegistry_setRespectedGameType_TestFail\|AnchorStateRegistry_updateRetirementTimestamp_Test\|AnchorStateRegistry_updateRetirementTimestamp_TestFail\|DelayedWETH_Hold_Test\|DelayedWETH_Initialize_Test\|DelayedWETH_Recover_Test\|DelayedWETH_Unlock_Test\|DelayedWETH_WithdrawFrom_Test\|DelayedWETH_Withdraw_Test\|DisputeGameFactory_Create_Test\|DisputeGameFactory_FindLatestGames_Test\|DisputeGameFactory_GetGameUUID_Test\|DisputeGameFactory_Owner_Test\|DisputeGameFactory_SetImplementation_Test\|DisputeGameFactory_SetInitBond_Test\|DisputeGameFactory_TransferOwnership_Test\|FaultDisputeGame_Test\|FaultDispute_1v1_Actors_Test\|PermissionedDisputeGame_Test\|SuperFaultDisputeGame_Test\|SuperFaultDispute_1v1_Actors_Test\|SuperPermissionedDisputeGame_Test"
```

Comment:
These are all the failures when trying to run against the latest optimism tests.